environments.yml

```diff
@@ -132,14 +132,19 @@ firefoxci:
       # No AMD families in northamerica-northeast1
       - zones: [northamerica-northeast1-a, northamerica-northeast1-b, northamerica-northeast1-c]
         families: [c4d, c3d]
+      - zones: [us-west1-a, us-west1-c]
+        families: [c3d]
+      # lssd c4d machines are not available in us-west1
+      - zones: [us-west1-a, us-west1-b]
+        families: [c4d]
+        suffixes: [lssd]
+      - zones: [us-west1]
```

Comment:
I'm not _super_ familiar with this code, but it looks like this is saying that `us-west1` as a whole is invalid?